command/head.go

```diff
@@ -289,7 +305,7 @@ func (m HeadBucketMessage) JSON() string {
 
 func validateHeadCommand(c *cli.Context) error {
 	if c.Args().Len() > 1 {
-		return fmt.Errorf("expected only 1 argument")
+		return fmt.Errorf("file or bucket name is required")
```

Comment:
since the target can be a remote object only, let's use the `object` word here.

Reply:
👍